.github/files/build-all-projects.sh

```diff
@@ -141,6 +141,22 @@ for SLUG in "${SLUGS[@]}"; do
 		cp -r "$BASE/.github/files/gh-autotagger/." "$BUILD_DIR/.github/."
 	fi
 
+	# Copy license.
+	LICENSE=$(jq -r '.license // ""' composer.json)
+	if [[ "$LICENSE" ]]; then
+		echo "License: $LICENSE"
+		if cp "$BASE/.github/licenses/$LICENSE.txt" "$BUILD_DIR/LICENSE.txt"; then
+			echo "License file copied."
+		else
+			echo "License value not approved."
```

Comment:
Flag the error for GH.
```suggestion
			echo "::error file=projects/$SLUG/composer.json::License value not approved."
```